lib/rules/no-duplicate-selectors/__tests__/index.mjs

```diff
@@ -546,6 +546,26 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [true, { ignoreSelectors: ['a'] }],
```

Comment:
[suggestion] Can you add some cases using a regex? E.g.

```js
{ ignoreSelectors: ['a', /foo/] }
```

Reply:
This guide for writing tests might be helpful: https://stylelint.io/developer-guide/rules#write-tests